pkgs/code_builder/lib/src/emitter.dart

```diff
@@ -437,15 +437,15 @@ class DartEmitter extends Object
     for (var a in spec.annotations) {
       visitAnnotation(a, output);
     }
+    if (spec.external) {
+      output.write('external ');
+    }
     if (spec.static) {
       output.write('static ');
     }
     if (spec.late && _useNullSafetySyntax) {
       output.write('late ');
```

Comment:
(There is no guard against being `external` and `late`, even though the two can never occur on the same valid declaration. Should there be a guard, or is it up the user to avoid specifying invalid combinations?)